### PR TITLE
Create Item production BOM and Item

### PR DESCRIPTION
**Pull request by KaviyaPeriyasamy**

### CODE REVIEW

File: apparelo/apparelo/doctype/item_production_detail/item_production_detail.json [hunk 1]
```diff
@@ -43,6 +37,18 @@
...
+   "fieldtype": "Link",
+   "label": "Item",
+   "options": "Item"
+  },
+  {
+   "fieldname": "name1",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Name"
+  },
```
Comment (karthikeyan5) [inline apparelo/apparelo/doctype/item_production_detail/item_production_detail.json]: Please remove this field and set autoname as "Prompt" instead.

File: apparelo/apparelo/doctype/item_production_detail/item_production_detail.json [hunk 2]
```diff
@@ -43,6 +37,18 @@
    "label": "Additional Flows",
    "options": "Item Production Detail Additional BOM"
   },
+  {
+   "fieldname": "item",
```
Comment (karthikeyan5) [inline apparelo/apparelo/doctype/item_production_detail/item_production_detail.json]: set this as mandatory?

**Review by karthikeyan5 — CHANGES_REQUESTED** [review]

**Comment by Vigneshsekar** [comment]
screenshot
screenshot


Shall we the same font size here?

**Comment by Vigneshsekar** [comment]
screenshot


Here too, we shall have the same font throughout the page...

**Review by karthikeyan5 — APPROVED** [review]

**Pull request merged**